### PR TITLE
podman v3 pod bindings

### PR DESCRIPTION
**Pull request by baude**
add options to each pod binding for future wiggle room

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM

File: pkg/bindings/pods/pods.go [hunk 1]
```diff
@@ -15,10 +13,14 @@ import (
...
 
-func CreatePodFromSpec(ctx context.Context, s *specgen.PodSpecGenerator) (*entities.PodCreateReport, error) {
+func CreatePodFromSpec(ctx context.Context, s *specgen.PodSpecGenerator, options *CreateOptions) (*entities.PodCreateReport, error) {
 	var (
 		pcr entities.PodCreateReport
 	)
+	if options == nil {
+		options = new(CreateOptions)
+	}
+	_ = options
```
Comment (Luap99) [inline pkg/bindings/pods/pods.go:23]: Why is this needed?

Comment (baude) [reply pkg/bindings/pods/pods.go:23]: keeps the linters happy
